### DISCUSSION TITLE
Linting

### DISCUSSION THREAD

**Discussion by Andrewnt219** [vercel/next.js#15503] [Help] [answered]
Hi everyone,

Does Next.js require any additional config in order to get eslint to work? I init an eslint config file (airbnb and react-hooks) at the root folder of my project but it wasn't working. I have always used create-react-app and these lintings come out of the box so not exactly sure I was doing it right either.

**Comment by jamesmosier** [comment, marked as answer]
Where are you wanting to see the lint output? In your terminal? Or in your editor (i.e. Visual Studio Code)? Or lint it manually by running a command?

If you want the lint errors to show in your terminal, you need to use something like `eslint-loader` similar to how it is done in this issue https://github.com/vercel/next.js/issues/9904

`next.config.js`

```js
module.exports = {
  webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
    if (dev) {
      config.module.rules.push({
        test: /\.(j|t)s?$/,
        exclude: /node_modules/,
        loader: 'eslint-loader',
      })
    }
    return config
  },
}
```

**Reply by Andrewnt219** [reply]
It works, thanks!

**Reply by Andrewnt219** [reply]
it worked with other eslint config, but as soon as I use extends: "react-app" from eslint-config-react-app, it stopped showing in terminal. Do you have any ideas why?
